CHANGELOG.md

```diff
@@ -59,8 +61,6 @@ BUG FIXES:
 * backends: Fix crash when interrupting during interactive prompt for values ([#36448](https://github.com/hashicorp/terraform/issues/36448))
 
 * Fixes hanging behavior seen when applying a saved plan with -auto-approve using the cloud backend ([#36453](https://github.com/hashicorp/terraform/issues/36453))
-
-
```

Comment:
This was [introduced when preparing the branch](https://github.com/hashicorp/terraform/pull/36639/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed), and is now being removed when cleaning up the branch.

I don't think it matters, but something to be aware of.